### PR TITLE
[Scala] add arange to Symbol and NDArray

### PR DESCRIPTION
**Pull request by Ldpe2G**

### CODE REVIEW

**Comment by Ldpe2G** [comment]
@javelinjs help to review ?

File: scala-package/core/src/main/scala/ml/dmlc/mxnet/DType.scala [hunk 1]
```diff
@@ -33,4 +33,11 @@ object DType extends Enumeration {
       case DType.Float64 => 8
     }
   }
+  private[mxnet] def getTypeStr(dtype: DType): String = dtype match {
```
Comment (javelinjs) [inline scala-package/core/src/main/scala/ml/dmlc/mxnet/DType.scala]: can make enum self-explanatory, e.g., `val Float32 = Value(0, "float32")`

Comment (Ldpe2G) [inline scala-package/core/src/main/scala/ml/dmlc/mxnet/DType.scala]: ok I will take a look.

**Comment by Ldpe2G** [comment]
@javelinjs Scala Test has passed successfully.

**Pull request merged**